check/setup.go

```diff
@@ -14,15 +14,22 @@ import (
 func (check *HealthCheck) connect(firstConnection bool, stop <-chan struct{}) error {
 	var createHealthTopicIfMissing = firstConnection
 	var createReplicationTopicIfMissing = firstConnection
+	var MaxRetries = 5
 	ticker := time.NewTicker(check.config.retryInterval)
 	defer ticker.Stop()
 	for {
 		select {
 		case <-ticker.C:
 			if err := check.tryConnectOnce(&createHealthTopicIfMissing, &createReplicationTopicIfMissing); err == nil {
 				return nil
+			}else{
```

Comment:
Thanks for making the changes to support multiple brokers. I found a major issue and I need your help to resolve it. If one of the brokers is down during the initialization, the service gets into an infinite loop to connect to the unavailable broker and the cluster response is overridden with **_{"status": "red"}_**. That gives a false impression that the overall cluster is down, whereas other brokers are up and running. If all brokers are up and running, then it gives the complete result.